### ISSUE TITLE
Feature request: Option to remove working directory

### ISSUE THREAD

**Issue by Somebodyisnobody** [Fulgen301/LCEdit#13]
When I move or remove a working directory from my disk I cannot remove the working dir in LCedit. An option to remove a wirkong dir would be useful.

**Comment by Fulgen301** [comment]
Indeed, the plugin should have gotten a better interface which I haven't gotten around to implement yet. As a workaround, you can remove working directories from the configuration file at `%appdata%\LCEdit\LCEdit.conf`.
